src/components/table/util.js

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+import { css } from 'theme-ui';
+
+const withTruncatedProp = (comp) => {
+  return styled(comp)(
+    ({ theme, color, bg, isTruncated, ...props }) =>
+      css({
+        color,
+        bg,
+        ...props,
+        ...(isTruncated ? {
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+          whiteSpace: 'nowrap',
+          maxWidth: '0px',
+        } : {}),
+      })(theme)
+  );
+};
+
+const getThemeStylesObj = (theme, styleObj, obj) => theme['styles'][styleObj][obj];
```

Comment:
- [ ] Do we need this helper? 
- [ ] If yes, then I suppose it should be a global helper. I see there is no way to place this helper under table utils?